packages/router/tsconfig.json

```diff
@@ -9,11 +9,6 @@
     }
   },
   "include": ["src", "./ambient.d.ts"],
-  "exclude": [
-    "dist",
-    "node_modules",
-    "**/__mocks__",
-  ],
```

Comment:
Might seem unrelated, because this TSConfig is outside of type tests folder. The problem is that `exclude` here overrides the ones from root config: 

https://github.com/redwoodjs/redwood/blob/a626553abcde3e3cbaf2e10c68dc8543698c9006/tsconfig.compilerOption.json#L20-L26

As you see, `**/*.test.*` is not excluded. Not sure if that was expected, but TSTyche picked up type test files from the `dist` folder. (`tsconfig.json` is missing there, so the test run failed.)

Reply:
If I remember correctly - the reason its this way is because we still build types with tsc (and we have to tell it to ignore the tests). 

Quite possibly the wrong way to do it though :)

Reply:
That looks right to my eye. The problem I try to point is different: the root `tsconfig.compilerOption.json` excludes test files nicely; but `packages/router/tsconfig.json` extends from the root TSConfig, overrides its `exclude` field and test files get included in the build.

My suggestion is to remove the override so that test files are excluded. That’s all is changing here.